src/content/guides/shimming.md

```diff
@@ -90,7 +90,17 @@ What we've essentially done here is tell webpack...
 If we run a build, we should still see the same output:
 
 ``` bash
-TODO: Include output
+./node_modules/.bin/webpack --config webpack.conf.js
+
```

Comment:
Can you drop these two lines (`93` and `94`)? Most of the guides stem from _Getting Started_ where setting up npm scripts (i.e. `npm run build`) is [recommended](https://webpack.js.org/guides/getting-started/#npm-scripts).

Reply:
Also, in most of the other output examples we don't include the actual command.

Reply:
Thank you for review, I copied it from another page mentioned in this guide :) Removing..